shared/chat/conversation/messages/popup.native.js

```diff
@@ -71,14 +75,16 @@ function MessagePopup (props: TextProps | AttachmentProps) {
     items = items.concat(_attachmentMessagePopupHelper(aProps))
   }
 
-  items.push({
-    danger: true,
-    onClick: () => {
-      onDeleteMessage(message)
-      onHidden()
-    },
-    title: 'Delete',
-  })
+  if (message.author === you) {
+    items.push({
+      danger: true,
+      onClick: () => {
+        onDeleteMessage(message)
+        onHidden()
```

Comment:
These `onHidden` callbacks... can they be refactored after your `onHidden` changes land?